rero_ils/static/scss/rero_ils/header.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2018 RERO.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+
+.rero-ils-logo {
+    max-width: 60px;
+}
+
+.navbar {
+    height: 100px;
+}
+
+header .input-group {
+  width: 100%;
+}
+
+.rero-ils-content-push-down {
+    height: 100px;
+}
+
+.rero-ils-top-menu {
+    font-size: 18px;
```

Comment:
why not em ?

Reply:
The bootstrap variable `$font-size-large` is precisely what we need. I made the change.